Gruntfile.js

```diff
@@ -856,9 +856,21 @@ module.exports = function(grunt) {
     },
     jsdoc : {
       dist : {
-          src: ['webapp/src/js/**/*.js', 'admin/src/js/**/*.js'],
+          src: [
+            'admin/src/js/*.js',
+            'admin/src/js/**/*.js',
+            'api/src/*.js',
+            'api/src/**/*.js',
+            'sentinel/src/*.js',
+            'sentinel/src/**/*.js',
+            'webapp/src/js/*.js',
+            'webapp/src/js/**/*.js'
+          ],
           options: {
-              destination: 'jsdoc-docs'
+              destination: 'jsdoc-docs',
+              configure: 'node_modules/angular-jsdoc/common/conf.json',
+              template: 'node_modules/angular-jsdoc/angular-template',
+              readme: './README.md'
```

Comment:
I think we should build these as separate docs. If you're writing code for api it's not helpful to know what's in admin. It also means we can only use the angular jsdoc config and templates in admin and webapp.

Reply:
Done. There might be a need to have a different template for the non-angular docs. I'm still looking into this.